src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form onsubmit="onSubmit()"
+          action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="POST"
+          class="welcome-form">
```

Comment:
3rd item of [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md), if there are more then 3 attributes, each of them should start on a new line
```suggestion
    <form 
      onsubmit="onSubmit()"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="POST"
      class="welcome-form"
    >
```